src/main/webapp/WEB-INF/jsp/home.jsp

```diff
@@ -19,17 +20,44 @@
 
 <body id="page-top">
   <%@include file="/WEB-INF/components/header.html"%>
-  <div class="container">
-    Home Page
-  </div>
+  <!-- Masthead-->
+  <header class="header-bg-img masthead text-white text-center">
+    <div class="container d-flex align-items-center flex-column">
+      <!-- Masthead Heading-->
+      <h1 class="masthead-heading text-uppercase mb-2">Deal Finder</h1>
+      <!-- Search Bar-->
+      <form class="form-inline active-pink-3 active-pink-4">
+        <i class="fas fa-search" aria-hidden="true"></i>
+        <input class="form-control form-control-sm ml-3 w-75" type="text" placeholder="Search" aria-label="Search">
+      </form>
+    </div>
+  </header>
+  <section class="page-section" id="popular-deals">
+    <h2 class="page-section-heading text-center text-uppercase text-secondary mb-0">Trending Deals</h2>
+    <%@include file="/WEB-INF/components/section-carousel.html"%>
```

Comment:
try to code the hardcoded data in a way that can be easily updated when the backend is made